k-distribution/tutorial/1_k/1_lambda/lesson_9/lambda.md

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2014-2019 K Team. All Rights Reserved.
+<!--- Copyright (c) 2014-2020 K Team. All Rights Reserved. --->
 
```

Comment:
Use the format used in other markdown files for the copyright instead. Look at `k-distribution/tutorial/1_k/README.md` for an example.

Reply:
I used the format from imp.md. I changed both files.